preprocessing/models/model5-estimator.js

```diff
@@ -65,34 +65,41 @@ function createMessage(id) {
   return Object.assign({}, aggregateOutput[id]);
 }
 
+// Returns the arsenic values
+function getArsenicValues(region, depth) {
+  let retval = {};
```

Comment:
don't need retval, use return in all the branches below

Reply:
This was changed in 0544d1d